client/src/pages/OrganisationProfile.js

```diff
@@ -201,6 +208,13 @@ const OrganisationProfile = () => {
     }
   };
 
+  const handleOrgDelete = () => {
```

Comment:
If you implement my other suggestion we need `e.preventDefault()` as it is now anchor tag
```suggestion
  const handleOrgDelete = (e) => {
    e.preventDefault();
```

Reply:
done